### ISSUE TITLE
How to auto open an album? 

### ISSUE THREAD

**Issue by venhels** [alanhamlett/jQuery-Picasa-Gallery#6]
Good day sir, How to set this plugin to open an album as the page start.

**Comment by maniekandan55** [comment]
Yeah. Me too looking for that. Hi Alan, you have praised us with a wonderful script, which is really awesome. This script currently displaying the ALBUMS as main page. Is it possible to set a particular album as main page.?
